Core/include/Acts/Vertexing/ImpactPointEstimator.hpp

```diff
@@ -209,6 +213,12 @@ class ImpactPointEstimator {
   /// Configuration object
   const Config m_cfg;
 
+  /// Logging instance
+  std::shared_ptr<const Logger> m_logger;
```

Comment:
I think this needs to be a shared pointer bc the `ImpactPointEstimator` is passed to both Finder and Fitter

Reply:
alternatively you could define a copy constructor which clones the logger